### DISCUSSION TITLE
How to install with opensuse?

### DISCUSSION THREAD

**Discussion by Dregen-Yor** [WiringPi/WiringPi#217] [Q&A] [answered]
How to build wiringpi in opensuse or in other rpm systems?

**Comment by mwallner** [comment, marked as answer]
Are you currently using OpenSUSE on a Raspberry Pi? While we primarily support Debian, you can still utilize the `./build` script. Just make sure you have the necessary build tools (such as gcc and make) installed on your system.

Please note that executing `./build` will not only generate the library but also copy the resulting artifacts to various locations on your system, such as `/usr` and `/local`. Make sure to backup the output of the script execution if you intend to uninstall the artifacts later on.
